src/olympia/devhub/urls.py

```diff
@@ -74,6 +74,8 @@
     url('^versions$', views.version_list, name='devhub.addons.versions'),
     url('^versions/delete$', views.version_delete,
         name='devhub.versions.delete'),
+    url('^versions/undisable$', views.version_undisable,
```

Comment:
I call it 'enable' in user facing text.   I don't particularly mind if its undisable, enable or restore - is there a precedent in Olympia code already?